packages/account/src/test-utils/launchNode.ts

```diff
@@ -104,6 +131,7 @@ function getFinalStateConfigJSON({ stateConfig, chainConfig }: SnapshotConfigs)
  * @param fuelCorePath - the path to the fuel-core binary. (optional, defaults to 'fuel-core')
  * @param loggingEnabled - whether the node should output logs. (optional, defaults to true)
  * @param basePath - the base path to use for the temporary folder. (optional, defaults to os.tmpdir())
+ * @param includeInitialState - whether to initialise the chain with some default initial state. (optional, defaults to false)
```

Comment:
Could this be a breaking change? Before we included state by default, granted the param isn't breaking but the change in expectation is? It may need to be default `true` with a deprecation notice?

Reply:
Yeah..good point. I'll default it to `true` for now

Reply:
Even if the default is `true`, then for this to be a non-breaking change, would we not need to match the current `stateConfig.json` file contents?

Reply:
IMO as long as there is some state it is fine

Reply:
While that's a good point @petertonysmith94, it's worth thinking if we can _ever_ be sure if changing the chain config or removing state from the default config can break something for people who might be using them for tests?

The only way we can really be sure is by providing all the state currently present like you said, and phasing it out via a deprecation notice like Dan mentioned.

Thoughts? @FuelLabs/sdk-ts

Reply:
Sorry for the lack of context here, but why are we adding this new flag `includeInitialState`? 

Is this a required change for this PR? Or can we make this change in a different PR?

Reply:
This new flag had to be added so that there are no breaking changes in the `fuels` CLI and `create fuels` since these tools need at least two wallets to be funded by default.